org.copyq.copyq.json

```diff
@@ -0,0 +1,34 @@
+{
+    "app-id": "org.copyq.copyq",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "copyq",
+
+    "tag": "v3.2.0",
+    "commit": "ac9f67ee99caa072e85951ba9b6b6f89c1e0ee39",
+
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--env=XDG_CURRENT_DESKTOP=''"
```

Comment:
`"--talk-name=org.kde.StatusNotifierWatcher"` should fix your tray icon. (The fact it depends upon this env var at all is broken, but that seems to be something to blame on a library you use)